doc/nrf/libraries/nfc/lib_nfc_t2t.rst

```diff
@@ -0,0 +1,17 @@
+.. _lib_nfc_t2t:
+
+Type 2 Tag
+##########
+
+Use the Type 2 Tag library in the :ref:`nrfxlib repository <nrfxlib:nfc>` to enable NFC Type 2 Tags and configure the tag data.
+:ref:`nrfxlib:type_2_tag` contains documentation for the library.
+
+See the :ref:`ug_nfc` user guide for information about :ref:`ug_nfc_programming`.
+
+The following Type 2 Tag libraries are provided by the |NCS|:
```

Comment:
```suggestion
The following Type 2 Tag libraries for Polling Device are provided by the |NCS|:
```
To clearly seperate T2T libraries for tag (nrfxlib/nfc) and poller (nrf/include/nfc/t2t). @grochu, any thoughts?

Reply:
Changed to: 
Additionally, the |NCS| provides libraries for reading Type 2 Tags.
Note that these require an NFC polling device.
The following libraries are provided: